optimize/client/src/modules/newReport.json

```diff
@@ -96,42 +96,6 @@
     "reportType": "process",
```

Comment:
here is one more `combined: false` above

Reply:
I cannot remove this just now, since the combined report is not yet removed from the backend. If I remove it, the evaluation will fail.

Reply:
then we have to wait for the BE changes, update this, and then we can merge

Reply:
I added a note to the [issue](https://github.com/camunda/camunda-optimize/issues/13439) to remove it later with the "reportType" field of the decision reports.